### DISCUSSION TITLE
Decommissioning with enabled tiering

### DISCUSSION THREAD

**Discussion by lmctv** [minio/minio#15424] [Q&A] [answered]
The question is simple: the missing support for decommissioning in presence of tiering
    https://docs.min.io/minio/baremetal/installation/decommission-pool.html#decommissioning-does-not-support-tiering
is just a temporary "glitch", which is expected to get fixed in the future, or is considered a permanent by-design feature?

**Comment by harshavardhana** [comment, marked as answer]
It may be addressed in future, we do not expect that you have multiple pools on the hot tier since it's just a metadata tier for the most part.

It may be implemented if we get some serious deployment requirement.

**Reply by kongslund** [reply]
> It may be addressed in future, we do not expect that you have multiple pools on the hot tier since it's just a metadata tier for the most part.
> 
> It may be implemented if we get some serious deployment requirement.

The servers that make up the hot tier will eventually reach end-of-life. It seems natural to be able to expand the hot tier by adding a server pool and then decommissioning the old server pool. Since that is currently not possible, which approaches do you recommend to complete a hardware upgrade cycle of the hot tier?

**Reply by harshavardhana** [reply]
Are you already at end of life for your hardware? And what is your deployment size? And how much of data is already tiered?

**Reply by kongslund** [reply]
It is not an issue for us right now — we're still planning. I am trying to make sure that a system design involving MinIO and tiering won't cause a major headache a few years from now.

**Reply by harshavardhana** [reply]
Well if you have a large enough use case and you are our customer - we are more than happy to put in resources to look at it. For now there is no real interest internally and we have other higher priorities.

This can be implemented before your hardware would reach end of life.
